libcloud/compute/drivers/softlayer.py

```diff
@@ -278,6 +278,8 @@ def create_node(self, **kwargs):
         :type       ex_os: ``str``
         :keyword    ex_keyname: The name of the key pair
         :type       ex_keyname: ``str``
+        :keyword    post_uri: Post-install script to run after reload
```

Comment:
It's an extension (provider specific / non-standard) argument, so please prefix it with `ex_`.